deployment/README.md

```diff
@@ -0,0 +1,92 @@
+# GCP Project Setup (WIP)
+
+Ensure the `GOOGLE_CLOUD_PROJECT` environment variable is exported for the project:
+```bash
+export GOOGLE_CLOUD_PROJECT=<PROJECT_ID>
+```
+The official project id of OSV is `oss-vdb`.
+
+## Terraform
+
+In this directory (`/deployment`):
+
+Initialise terraform:
+```bash
+terraform init
+```
+
+Plan and apply required project infastructure:
+```bash
+terraform plan
+```
+```bash
+terraform apply
+```
+Use these commands to deploy any configuration changes.
+Inspect the proposed changes when running the command to see what resources will
+be added, modified and destroyed.
+
+## Setting up auto-scaler
+
+There doesn't seem to be a good way to set this up within terraform.
+
+The following instructions are from [here](https://cloud.google.com/kubernetes-engine/docs/tutorials/external-metrics-autoscaling#step1).
+
+```bash
+kubectl create clusterrolebinding cluster-admin-binding \
+    --clusterrole cluster-admin --user "$(gcloud config get-value account)"
+```
+
+```bash
+kubectl apply -f https://raw.githubusercontent.com/GoogleCloudPlatform/k8s-stackdriver/master/custom-metrics-stackdriver-adapter/deploy/production/adapter_new_resource_model.yaml
+```
+
+## Submit local builds to container registry
+
+Currently, only worker is implemented.
+
+If `GOOGLE_CLOUD_PROJECT` is not set, the project ID will default to `oss-vdb`.
+
+Inside `docker/worker-base/`
+```bash
+./build.sh <TAG>
+```
+
+Then, inside `docker/worker/`
+```bash
+./build.sh <TAG>
+```
+
+## Deploy builds to cluster
+
+First, edit the image path in `deployment/oss-vdb-test/cloudbuild/gke/workers/workers.yaml` to match the project ID:
```

Comment:
Can you add a TODO to use https://kubernetes.io/docs/tasks/manage-kubernetes-objects/kustomization/ ?